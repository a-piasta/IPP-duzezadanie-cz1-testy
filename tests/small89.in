addRoad;sbpaxxvwxainxzvxjtwx;sbpaxxvwxainxzvxjtwx;25;-4
extendRoute;50;cqabcyncgalvmaaphjqnqfcvfdln
repairRoad;iyiyzowylcqafrnp;sbpaxxvwxainxzvxjtwx;5
newRoute;946;xgnjnkpedhhzdhnncabqfnv;xgnjnkpedhhzdhnncabqfnv
addRoad;sbpaxxvwxainxzvxjtwx;iyiyzowylcqafrnp;3;-6
getRouteDescription;946
getRouteDescription;946
removeRoad;sbpaxxvwxainxzvxjtwx;iyiyzowylcqafrnp
addRoad;zmoutshglnzgnsgn;grvytyatlcanpvzpxqxig;29;11
getRouteDescription;995
newRoute;435;iyiyzowylcqafrnp;zmoutshglnzgnsgn
newRoute;320;zmoutshglnzgnsgn;grvytyatlcanpvzpxqxig
removeRoad;grvytyatlcanpvzpxqxig;zmoutshglnzgnsgn
extendRoute;435;zmoutshglnzgnsgn
getRouteDescription;435
extendRoute;946;cqabcyncgalvmaaphjqnqfcvfdln
repairRoad;zmoutshglnzgnsgn;grvytyatlcanpvzpxqxig;-8
removeRoad;iyiyzowylcqafrnp;sbpaxxvwxainxzvxjtwx
newRoute;972;veprsauwaoitklnfmdwmvwzkw;cqabcyncgalvmaaphjqnqfcvfdln
repairRoad;zmoutshglnzgnsgn;cqabcyncgalvmaaphjqnqfcvfdln;1
getRouteDescription;630
addRoad;uiwmlbdysksubstbpraqznpyinjdj;uiwmlbdysksubstbpraqznpyinjdj;25;0
newRoute;308;uiwmlbdysksubstbpraqznpyinjdj;cqabcyncgalvmaaphjqnqfcvfdln
repairRoad;xgnjnkpedhhzdhnncabqfnv;iyiyzowylcqafrnp;-1
extendRoute;320;sbpaxxvwxainxzvxjtwx
removeRoad;sbpaxxvwxainxzvxjtwx;iyiyzowylcqafrnp
removeRoad;zmoutshglnzgnsgn;grvytyatlcanpvzpxqxig
repairRoad;grvytyatlcanpvzpxqxig;zmoutshglnzgnsgn;6
repairRoad;xgnjnkpedhhzdhnncabqfnv;cqabcyncgalvmaaphjqnqfcvfdln;14
extendRoute;320;sbpaxxvwxainxzvxjtwx
addRoad;xgnjnkpedhhzdhnncabqfnv;cqabcyncgalvmaaphjqnqfcvfdln;17;8
addRoad;uiwmlbdysksubstbpraqznpyinjdj;zmoutshglnzgnsgn;19;-9
newRoute;484;xtalxnbrilmpgud;grvytyatlcanpvzpxqxig
newRoute;343;zmoutshglnzgnsgn;uiwmlbdysksubstbpraqznpyinjdj
getRouteDescription;946
newRoute;254;sbpaxxvwxainxzvxjtwx;uiwmlbdysksubstbpraqznpyinjdj
addRoad;iyiyzowylcqafrnp;uiwmlbdysksubstbpraqznpyinjdj;3;13
removeRoad;cqabcyncgalvmaaphjqnqfcvfdln;xgnjnkpedhhzdhnncabqfnv
extendRoute;320;ozexzwespczkkpfkjfvgtxzdkov
repairRoad;iyiyzowylcqafrnp;sbpaxxvwxainxzvxjtwx;9
repairRoad;veprsauwaoitklnfmdwmvwzkw;cqabcyncgalvmaaphjqnqfcvfdln;0
extendRoute;320;xgnjnkpedhhzdhnncabqfnv
extendRoute;435;uiwmlbdysksubstbpraqznpyinjdj
getRouteDescription;308
getRouteDescription;343
addRoad;zmoutshglnzgnsgn;ozexzwespczkkpfkjfvgtxzdkov;25;-1
removeRoad;uiwmlbdysksubstbpraqznpyinjdj;iyiyzowylcqafrnp
removeRoad;zmoutshglnzgnsgn;ozexzwespczkkpfkjfvgtxzdkov
repairRoad;uiwmlbdysksubstbpraqznpyinjdj;zmoutshglnzgnsgn;6
repairRoad;iyiyzowylcqafrnp;cqabcyncgalvmaaphjqnqfcvfdln;-11
newRoute;51;cqabcyncgalvmaaphjqnqfcvfdln;grvytyatlcanpvzpxqxig
getRouteDescription;51
getRouteDescription;320
repairRoad;xgnjnkpedhhzdhnncabqfnv;xgnjnkpedhhzdhnncabqfnv;11
repairRoad;iyiyzowylcqafrnp;grvytyatlcanpvzpxqxig;-12
addRoad;grvytyatlcanpvzpxqxig;grvytyatlcanpvzpxqxig;8;1
newRoute;853;uiwmlbdysksubstbpraqznpyinjdj;ozexzwespczkkpfkjfvgtxzdkov
repairRoad;veprsauwaoitklnfmdwmvwzkw;grvytyatlcanpvzpxqxig;-4
