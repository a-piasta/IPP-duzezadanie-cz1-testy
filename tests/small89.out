1: NIE
2: NIE
3: NIE
4: NIE
5: TAK
6: 
7: 
8: TAK
9: TAK
10: 
11: NIE
12: TAK
13: NIE
14: NIE
15: 
16: NIE
17: NIE
18: NIE
19: NIE
20: NIE
21: 
22: NIE
23: NIE
24: NIE
25: NIE
26: NIE
27: NIE
28: NIE
29: NIE
30: NIE
31: TAK
32: TAK
33: NIE
34: TAK
35: 
36: NIE
37: TAK
38: TAK
39: NIE
40: NIE
41: NIE
42: NIE
43: NIE
44: 
45: 343;zmoutshglnzgnsgn;19;-9;uiwmlbdysksubstbpraqznpyinjdj
46: TAK
47: TAK
48: TAK
49: TAK
50: NIE
51: NIE
52: 
53: 320;zmoutshglnzgnsgn;29;11;grvytyatlcanpvzpxqxig
54: NIE
55: NIE
56: NIE
57: NIE
58: NIE
